qiskit_experiments/library/randomized_benchmarking/rb_analysis.py

```diff
@@ -184,39 +194,27 @@ def _extra_database_entry(self, fit_data: curve.FitData) -> List[AnalysisResultD
         )
 
         # Calculate EPG
-        if not self.options.gate_error_ratio:
-            # we attempt to get the ratio from the backend properties
-            if not self.options.error_dict:
-                gate_error_ratio = RBUtils.get_error_dict_from_backend(
-                    backend=self._backend, qubits=self._physical_qubits
-                )
-            else:
-                gate_error_ratio = self.options.error_dict
-        else:
-            gate_error_ratio = self.options.gate_error_ratio
 
-        count_ops = []
-        for meta in self._data(label="raw_data").metadata:
-            count_ops += meta.get("count_ops", [])
-
-        if len(count_ops) > 0 and gate_error_ratio is not None:
-            gates_per_clifford = RBUtils.gates_per_clifford(count_ops)
+        if (
+            self.options.gate_counts_per_clifford is not None
+            and self.options.gate_error_ratio != "skip"
```

Comment:
Looking at below code it doesn't seem like it, since you use None to check that the option wasn't set at all

Reply:
This is correct because past experiment data entry doesn't have this value and initially it is `None`, but it is computed with `RBUtils.get_error_dict_from_backend` as before. "skip" is new option that doesn't exist before (if you don't need EPG values you can speed up the transpile).